build.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+set -e
+
+if [ ! -d "node_modules/.bin" ]; then
+  echo "Be sure to run \`npm install\` before building graphiql-workspace."
+  exit 1
+fi
+
+rm -rf dist/ && mkdir -p dist/
+babel src --out-dir dist/
+echo "Bundling graphiql-workspace.js..."
+browserify -g browserify-shim -s GraphiQL dist/index.js > graphiql-workspace.js
```

Comment:
I have renamed the export to use `graphiqlWorkspace` but I dont see any issue right now by using the standalone option. My understanding is it will export the whole export object and the name being passed is merely used for naming the global export. So you can do `graphiqlWorkspace.GraphiQLWorkspace` and `graphiqlWorkspace.AppConfig`.

Am i missing something ?